src/App.tsx

```diff
@@ -17,18 +49,29 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                query={query}
+                status={status}
+              />
             </div>
 
             <div className="block">
-              <Loader />
-              <TodoList />
+              {todos.length > 0 ? (
```

Comment:
You can shorten it, also will work
```suggestion
              {todos.length  ? (
```